persistent/Database/Persist/TH/Internal.hs

```diff
@@ -1079,17 +1081,66 @@ data MkPersistSettings = MkPersistSettings
     , mpsPrefixFields :: Bool
     -- ^ Prefix field names with the model name. Default: True.
     --
-    -- Note: this field is deprecated. Use the mpsFieldLabelModifier  and
+    -- Note: this field is deprecated. Use the 'mpsFieldLabelModifier'  and
     -- 'mpsConstraintLabelModifier' instead.
     , mpsFieldLabelModifier :: Text -> Text -> Text
-    -- ^ Customise the field accessors and lens names using the entity and field
-    -- name.  Both arguments are upper cased.
+    -- ^ Customise the field names (and lens names) for generated entity data types.
     --
-    -- Default: appends entity and field.
+    -- Default: simply appends entity name and field name, equivalent to
```

Comment:
```suggestion
    -- Default: appends entity name and field name, equivalent to
```

I try to avoid judgmental terms like `simply` when I'm thinking about it 😅